buildScripts/package.json

```
@@ -5,13 +5,13 @@
     "private": true,
     "license": "Apache-2.0",
     "devDependencies": {
-        "assert": "^1.4.1",
-        "clean-modules": "^1.0.4",
+        "assert": "^2.0.0",
+        "clean-modules": "^3.0.2",
         "command-line-args": "^5.0.2",
-        "command-line-usage": "^5.0.5",
-        "compare-versions": "^3.4.0",
+        "command-line-usage": "^7.0.1",
+        "compare-versions": "^6.0.0-rc.2",
```

Let's avoid using release candidates when possible
```suggestion
        "compare-versions": "^6.0.0",
```